app/lib/webpack/create-chain.js

```diff
@@ -211,7 +211,7 @@ module.exports = function (cfg, configName) {
       .options({
         esModule: false,
         limit: 10000,
-        name: `img/[name]${fileHash}.[ext]`
+        name: `img/[path][name]${fileHash}.[ext]`
```

Comment:
Hi,

Thanks for contributing!
Can you add `[path]` for the fonts too pls? (It's right below the images section)